okhttp/src/main/java/com/squareup/okhttp/internal/spdy/SpdyStream.java

```diff
@@ -522,8 +527,8 @@ private void emitDataFrame(boolean outFinished) throws IOException {
       }
       while (sendBuffer.size() > 0) {
         emitDataFrame(false);
+        connection.flush();
```

Comment:
This change is not strictly necessary, but unlikely to cause problems. If we don't flush between frames there's a risk of deadlock.